### PR TITLE
Extend search

### PR DESCRIPTION
**Pull request by Kathrin-Huber**
Depending on #2504

### CODE REVIEW

File: Kitodo/src/test/java/org/kitodo/production/services/data/ProcessServiceIT.java [hunk 1]
```diff
@@ -162,19 +162,19 @@ public void shouldFindByMetadata() throws DataException {
...
+    public void shouldFindByMetadataContent() throws DataException {
+        assertEquals(processNotFound, 1, processService.findByAnything("SecondMetaShort").size());
     }
 
     @Test
-    public void shouldNotFindByMetadata() throws DataException {
-        assertEquals("Process was found in index!", 0,
-            processService.findByMetadata(Collections.singletonMap("TSL_ATS", "Nope")).size());
+    public void shouldNotFindByMetadataContent() throws DataException {
+        assertEquals(processNotFound, 0, processService.findByAnything("Nope").size());
```
Comment (Beacze) [inline Kitodo/src/test/java/org/kitodo/production/services/data/ProcessServiceIT.java]: Will it really test metadata? Maybe test should be called differently if it calls findByAnything?

File: Kitodo/src/test/java/org/kitodo/production/services/data/ProcessServiceIT.java [hunk 2]
```diff
@@ -162,19 +162,19 @@ public void shouldFindByMetadata() throws DataException {
...
+    public void shouldNotFindByMetadata() throws DataException {
+        assertEquals("Process was found in index!", 0,
+                processService.findByMetadata(Collections.singletonMap("TSL_ATS", "Nope")).size());
     }
 
     @Test
-    public void shouldNotFindByMetadataContent() throws DataException {
-        assertEquals(processNotFound, 0, processService.findByMetadataContent("Nope").size());
+    public void shouldFindByMetadataContent() throws DataException {
+        assertEquals(processNotFound, 1, processService.findByAnything("SecondMetaShort").size());
```
Comment (Beacze) [inline Kitodo/src/test/java/org/kitodo/production/services/data/ProcessServiceIT.java]: Will it really test metadata? Maybe test should be called differently if it calls findByAnything?

File: Kitodo/src/test/java/org/kitodo/production/services/data/ProcessServiceIT.java (hunk 1 above)
Comment (Kathrin-Huber) [reply Kitodo/src/test/java/org/kitodo/production/services/data/ProcessServiceIT.java]: I changed the method name.

File: Kitodo/src/test/java/org/kitodo/production/services/data/ProcessServiceIT.java (hunk 2 above)
Comment (Kathrin-Huber) [reply Kitodo/src/test/java/org/kitodo/production/services/data/ProcessServiceIT.java]: Here it actually tests to find a Process by Metadatacontent

**Pull request merged**